lib/rex/ui/text/dispatcher_shell.rb

```diff
@@ -650,7 +654,14 @@ def shellsplitex(line)
     quote = nil
```

Comment:
What are your thoughts on updating the method comment above with a link to were this code was originally taken from? 👀

Reply:
Yea, I think that would be a good idea 👍